entry.sh

```diff
@@ -653,11 +679,8 @@ touch "${CERTS}/.ready"
 # remove lock
 remove_update_lock
 
-# exit-restart if near expiration (LetEncrypt only)
 while true; do
-    if ! check_pub_cert_expiry "${EXPORT_CERT_CHAIN_PATH}"; then exit; fi
-
+    check_cert_expiry "${EXPORT_CERT_CHAIN_PATH}"
```

Comment:
How does this condition cause the script to exit (and the container to restart) if the certificate is expired (check_cert_expiry returns 1)?

Reply:
it doesn't, there is a static 1d sleep before the container exits and restarts.

Reply:
If any of the certificates are expiring, `check_cert_expiry` will return `false` and the script will exit (because of `set -e`). The restart policy on the container will restart it and it will go around the loop again.

Reply:
I see. That makes sense why you removed the "if" statement that exits the script.  No other questions and everything else looks good.